finat/physically_mapped.py

```diff
@@ -125,6 +139,12 @@ def point_evaluation(self, order, refcoords, entity=None):
         raise NotImplementedError("TODO: not yet thought about it")
 
 
+class DirectlyDefinedElement(NeedsCoordinateMappingElement):
+    """Base class for directly defined elements such as direct
+    serendipity that bypass a coordinate mapping."""
+    pass
```

Comment:
So this is just a marker class.  (It defines no useful interface, merely marks that the derived class is of a certain type.)  Fine.